vignettes/data-handling.Rmd

```diff
@@ -27,19 +27,27 @@ library(esqlabsR)
 
 The workflow of modeling and simulation involves running simulations and comparing simulated data against observed data. Functionalities of `esqlabsR` require observed data to be present as [`DataSet`](https://www.open-systems-pharmacology.org/OSPSuite-R/reference/DataSet.html) objects. Please refer to the article [Observed data](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html) for information on how to load data from Excel or `*.pkml` files.
```

Comment:
Is the link here supposed to be [https://www.open-systems-pharmacology.org/OSPSuite-R/dev/articles/observed-data.html](url)?

Reply:
True, will fix.